README.md

```diff
@@ -73,92 +87,35 @@ Sims.jl has one main module named `Sims` and the following submodules:
 Basic example
 -------------
 
-Sims defines a basic symbolic class used for unknown variables in
-the model. As unknown variables are evaluated, expressions (of
-type `MExpr`) are built up.
-
-``` julia
-julia> using Sims
-
-julia> a = Unknown()
-##1243
-
-julia> a * (a + 1)
-MExpr(*(##1243,+(##1243,1)))
-```
-
+Sims uses ModelingToolkit to build up models. All equations use the
+ModelingToolkit variables and syntax.
 In a simulation, the unknowns are to be solved based on a set of
 equations. Equations are built from device models. 
 
 A device model is a function that returns a vector of equations or
-other devices that also return lists of equations. The equations
-each are assumed equal to zero. So,
-
-``` julia
-der(y) = x + 1
-```
-
-Should be entered as:
-
-``` julia
-der(y) - (x+1)
-```
-
-`der` indicates a derivative.
+other devices that also return lists of equations. 
 
 The Van Der Pol oscillator is a simple problem with two equations
 and two unknowns:
 
 ``` julia
 function Vanderpol()
-    y = Unknown(1.0, "y")   # The 1.0 is the initial value. "y" is for plotting.
-    x = Unknown("x")        # The initial value is zero if not given.
+    @variables x(t) y(t)
```

Comment:
```suggestion
    @variables x(t) y(t) = 1.0
```

can be allowed now that there's the metadata @yingboma?

Reply:
Interesting. A knew metadata was coming, but I haven't seen examples. Initial values in variables would be great.

Reply:
Any hints on how to programatically create a variable that has an initial value? I am currently using `Num(Variable{ModelingToolkit.FnType{Tuple{Any},Real}}(gensym("i")))(t)`, and I also see `setmetadata`.

Reply:
We want to allow interpolating into the macros, i.e . `x = @variables $y = 1.0`